ui/app/src/components/AceEditor/AceEditor.tsx

```diff
@@ -253,6 +253,7 @@ function AceEditorComp(props: AceEditorProps, ref: MutableRef<AceAjax.Editor>) {
             refs.current.elem.appendChild(pre);
             // @ts-ignore - Ace types are incorrect; they don't implement the constructor that receives options.
             aceEditor = ace.edit(pre, refs.current.options);
+            aceEditor.setFontSize('14px');
```

Comment:
Why not `18px`?

Reply:
Please see [comment on the ticket](https://github.com/craftercms/craftercms/issues/7210#issuecomment-2315596621).